### DISCUSSION TITLE
Asciidoc code field syntax highlighting

### DISCUSSION THREAD

**Discussion by jimboolio** [directus/directus#16894] [Feature Requests]
Hello!

Could you add AsciiDoc as supported code field language? It is quite popular markup format and highlight js supports it out of the box, I saw some references to that in the app source but I didn't spend enough time to understand how to enable missing one if that is even the correct file I was reading.

If the highlighter library is something other than highlight js, could someone point me to the correct library where the highlight support should be implemented?

One extremely awesome feature would be to add separate editor like there is for markdown but for asciidoc, but that would be somewhat time consuming to develop and basic highlighting support would be quite an improvement alone

**Comment by rijkvanzanten** [comment]
We're using CodeMirror for the Code interface: https://github.com/directus/directus/blob/40f94e6907d2999341546838720cf3086653515b/app/src/interfaces/input-code/input-code.vue. The supported languages are currently imported from here: https://github.com/directus/directus/blob/40f94e6907d2999341546838720cf3086653515b/app/src/interfaces/input-code/import-codemirror-mode.ts. I don't think CodeMirror natively supports asciidoc 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
